lib/internal/modules/esm/loader.js

```diff
@@ -470,9 +480,15 @@ class ModuleLoader {
    * @returns {Promise<ModuleExports>}
    */
   async import(specifier, parentURL, importAttributes, isEntryPoint = false) {
-    const moduleJob = await this.getModuleJob(specifier, parentURL, importAttributes);
-    const { module } = await moduleJob.run(isEntryPoint);
-    return module.getNamespace();
+    return onImport.tracePromise(async () => {
+      const moduleJob = await this.getModuleJob(specifier, parentURL, importAttributes);
```

Comment:
Should we include the `importAttributes` to the event?

Reply:
Perhaps in a follow-up PR?